src/httpserver.cpp

```diff
@@ -345,7 +345,7 @@ static bool HTTPBindAddresses(struct evhttp* http)
 
     // Bind addresses
     for (std::vector<std::pair<std::string, uint16_t> >::iterator i = endpoints.begin(); i != endpoints.end(); ++i) {
-        LogPrint("http", "Binding RPC on address %s port %i\n", i->first, i->second);
+        LogPrint("Binding RPC on address %s port %i\n", i->first, i->second);
```

Comment:
See line 353 here; I think that approach will work. If you force push a commit, I'll enable CI and run the tests myself too.